routes/v1.js

```diff
@@ -79,6 +80,77 @@ router.get('/siteinfo/:uri/:prop?', function(req, res) {
 });
 
 
+/****************************
+ *  PAGE MASSAGING SECTION  *
+ ****************************/
+
+/**
+ * A helper function that obtains the HTML form enwiki and
+ * loads it into a domino DOM document instance.
+ *
+ * @param {String} title the title of the page to get
+ * @return {Promise} a promise resolving as the HTML element object
+ */
+function getBody(title) {
+
+    // get the page from enwiki
+    return preq.get({
+        uri: 'http://en.wikipedia.org/w/index.php',
+        qs: {
+            title: title
+        },
+        useQuerystring: true
```

Comment:
According to the [request docs](https://github.com/request/request#requestoptions-callback), it's a feature, not a bug :)